src/pid.cpp

```diff
@@ -299,15 +306,16 @@ double Pid::computeCommand(double error, ros::Duration dt)
 {
 
   if (dt == ros::Duration(0.0) || std::isnan(error) || std::isinf(error))
-    return 0.0;
+    return cmd_;
```

Comment:
What does this improve? It is unclear what a PID command should be in any of the cases in the `if` statement. The only reason we aren't throwing an exception here is to keep a simplistic, controller-friendly interface. We could change the return to `-inf` or `nan` if you wish ;)